### PR TITLE
[portchannel] Added ACL/PBH binding checks to the port before getting added to portchannel

### PR DESCRIPTION
**Pull request by vivekreddynv**
Signed-off-by: Vivek Reddy Karri <vkarri@nvidia.com>

#### What I did

The change to handle it in oA is already added. 

When this check is not performed when adding the config, the portchannel configuration will be inconsistent b/w Kernel and ASIC

#### How I did it

Utilize the match infra to implement methods to check for ACL/PBH bindings to a port

#### How to verify it
Unit tests

```
Add Config
root@r-lionfish-16:/home/admin# config pbh table add NVGRE --interface-list "Ethernet0,Ethernet56" --description "Whatever"
root@r-lionfish-16:/home/admin# sudo config acl add table EVERFLOW MIRROR --ports "Ethernet236"
root@r-lionfish-16:/home/admin# config portchannel add PortChannel1111

Try adding members to portchannel
root@r-lionfish-16:/home/admin# config portchannel member add PortChannel1111 Ethernet0
Usage: config portchannel member add [OPTIONS] <portchannel_name> <port_name>
Try "config portchannel member add -h" for help.

Error: Port Ethernet0 is already bound to following PBH_TABLES: ['PBH_TABLE|NVGRE']

root@r-lionfish-16:/home/admin# config portchannel member add PortChannel1111 Ethernet236
Usage: config portchannel member add [OPTIONS] <portchannel_name> <port_name>
Try "config portchannel member add -h" for help.

Error: Port Ethernet236 is already bound to following ACL_TABLES: ['ACL_TABLE|EVERFLOW']

Check debug dump:
root@r-lionfish-16:/home/admin# dump state portchannel_member "PortChannel1111|Ethernet0,PortChannel1111|Ethernet236"
{
    "PortChannel1111|Ethernet0": {
        "CONFIG_DB": {
            "keys": [],
            "tables_not_found": [
                "PORTCHANNEL_MEMBER"
            ]
        },
        "APPL_DB": {
            "keys": [],
            "tables_not_found": [
                "LAG_MEMBER_TABLE"
            ]
        },
        "ASIC_DB": {
            "keys": [],
            "tables_not_found": [
                "ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER"
            ]
        },
        "STATE_DB": {
            "keys": [],
            "tables_not_found": [
                "LAG_MEMBER_TABLE"
            ]
        }
    },
    "PortChannel1111|Ethernet236": {
        "CONFIG_DB": {
            "keys": [],
            "tables_not_found": [
                "PORTCHANNEL_MEMBER"
            ]
        },
        "APPL_DB": {
            "keys": [],
            "tables_not_found": [
                "LAG_MEMBER_TABLE"
            ]
        },
        "ASIC_DB": {
            "keys": [],
            "tables_not_found": [
                "ASIC_STATE:SAI_OBJECT_TYPE_LAG_MEMBER"
            ]
        },
        "STATE_DB": {
            "keys": [],
            "tables_not_found": [
                "LAG_MEMBER_TABLE"
            ]
        }
    }
}
root@r-lionfish-16:/home/admin# dump state portchannel "PortChannel1111"
{
    "PortChannel1111": {
        "CONFIG_DB": {
            "keys": [
                {
                    "PORTCHANNEL|PortChannel1111": {
                        "admin_status": "up",
                        "lacp_key": "auto",
                        "min_links": "1",
                        "mtu": "9100"
                    }
                }
            ],
            "tables_not_found": []
        },
        "APPL_DB": {
            "keys": [
                {
                    "LAG_TABLE:PortChannel1111": {
                        "admin_status": "up",
                        "mtu": "9100",
                        "oper_status": "down"
                    }
                }
            ],
            "tables_not_found": []
        },
        "ASIC_DB": {
            "keys": [],
            "tables_not_found": [
                "ASIC_STATE:SAI_OBJECT_TYPE_LAG"
            ]
        },
        "STATE_DB": {
            "keys": [
                {
                    "LAG_TABLE|PortChannel1111": {
                        "admin_status": "up",
                        "mtu": "9100",
                        "oper_status": "down",
                        "runner.active": "true",
                        "runner.fallback": "false",
                        "runner.fast_rate": "false",
                        "setup.kernel_team_mode_name": "loadbalance",
                        "setup.pid": "154",
                        "state": "ok",
                        "team_device.ifinfo.dev_addr": "1c:34:da:1c:9f:00",
                        "team_device.ifinfo.ifindex": "136"
                    }
                }
            ],
            "tables_not_found": []
        }
    }
}

```

### CODE REVIEW

**Review by dgsudharsan — APPROVED** [review]

**Review by dgsudharsan — DISMISSED** [review]

**Comment by liat-grozovik** [comment]
@nazariig please review

**Review by nazariig — APPROVED** [review]

File: utilities_common/db.py [hunk 1]
```diff
@@ -15,15 +15,15 @@ def __init__(self):
         self.db = SonicV2Connector(host="127.0.0.1")
 
         # Skip connecting to chassis databases in line cards
-        db_list = list(self.db.get_db_list())
+        self.db_list = list(self.db.get_db_list())
         if not device_info.is_supervisor():
             try:
                 db_list.remove('CHASSIS_APP_DB')
                 db_list.remove('CHASSIS_STATE_DB')
```
Comment (bingwang-ms) [inline utilities_common/db.py:22]: Should it be 
```suggestion
                self.db_list.remove('CHASSIS_APP_DB')
                self.db_list.remove('CHASSIS_STATE_DB')
```
?

**Review by bingwang-ms — APPROVED** [review]

**Review by dgsudharsan — APPROVED** [review]

**Comment by judyjoseph** [comment]
Please raise a new PR for 202111 ,, as there are conflicts

**Comment by vivekreddynv** [comment]
> Please raise a new PR for 202111 ,, as there are conflicts

Backport PR: https://github.com/Azure/sonic-utilities/pull/2186

**Pull request merged**